src/test_application/main.cpp

```diff
@@ -309,13 +317,61 @@ void debug_print_help()
         << "* flight    - Create an ActiveArea and start Magnum\n"
         << "\n"
         << "Other things to type:\n"
+        << "* list_pkg  - List Packages and Resources\n"
         << "* list_uni  - List Satellites in the universe\n"
         << "* list_ent  - List Entities in active scene\n"
         << "* list_upd  - List Update order from active scene\n"
         << "* help      - Show this again\n"
         << "* exit      - Deallocate everything and return memory to OS\n";
 }
 
+template <typename RES_T>
+void debug_print_resource_group(osp::Package& rPkg)
+{
+    osp::Package::group_t<RES_T> const &group = rPkg.group_get<RES_T>();
+
+    if (group.empty())
+    {
+        return;
+    }
+
+    std::cout << "  * TYPE: " << entt::type_name<RES_T>().value() << "\n";
+
+    for (auto const& [key, resource] : group)
+    {
+        std::cout << "    * " << (resource.m_data.has_value() ? "LOADED" : "RESERVED") << ": " << key << "\n";
```

Comment:
Now that I think of it, the reference count can be printed here too.

Also, sample output:
```
* PACKAGE: lzdb
  * TYPE: osp::PrototypePart
    * LOADED: part_phCapsule
    * LOADED: part_phEngine
    * LOADED: part_phFuselage
    * LOADED: part_phLinRCS
    * LOADED: part_spamcan
    * LOADED: part_stomper
  * TYPE: Magnum::Trade::ImageData<2>
    * LOADED: Material Metallic
    * LOADED: OSPData/adera/noise1024.png
    * LOADED: OSPData/adera/noise256.png
    * LOADED: ph_base
    * LOADED: spamcanhd
    * LOADED: stomper
    * LOADED: stomper_metal-stomper_rough
  * TYPE: Magnum::Trade::MeshData
    * LOADED: OSPData/adera/ph_capsule:Cylinder.001
    * LOADED: OSPData/adera/ph_engine:Cylinder.002
    * LOADED: OSPData/adera/ph_fuselage:Cylinder
    * LOADED: OSPData/adera/ph_plume:Cylinder.003
    * LOADED: OSPData/adera/ph_rcs:Cube
    * LOADED: OSPData/adera/ph_rcs_plume:Cylinder.005
    * LOADED: OSPData/adera/spamcan:Cylinder.001
    * LOADED: OSPData/adera/stomper:Cylinder.002
  * TYPE: adera::active::machines::ShipResourceType
    * LOADED: fuel

```